app/index.js

```diff
@@ -2,13 +2,12 @@
 
 var yeoman = require('yeoman-generator');
 var path = require('path');
-var fs = require('fs');
-var sortedObject = require("sorted-object");
+var sortedObject = require('sorted-object');
 
 var _ = require('underscore');
 
 module.exports = yeoman.generators.Base.extend({
-  init: function () {
+  initializing: function () {
     var notEmpty = function (str) {
```

Comment:
Can we rename this function to something explicit? `notEmpty` sounds like a boolean check, not an array filtering method...

Maybe `stringArray` or `arrayFromString`